packages/components/src/components/button-toggle-menu/button-highlight.jsx

```diff
@@ -3,8 +3,8 @@ import React     from 'react';
 
 const Highlight = ({ left, width }) => {
     const border_radius_size = '4px';
-    const left_offset        = (left < 110) ? 0 : left;
-    const width_offset       = (width < 110) ? 111 : width;
+    const left_offset        = (left < 60) ? 0 : left;
```

Comment:
have you checked in the trader for this styling, better override the styling on the usage rather than the component itself

Reply:
@mustofa-binary I checked the usage of this component in the `trader` and it was ok. 
I can't pass this data from usage place because  these are some of its internal states. they doesn't have external interface from `<button-toggle-menu />` (the component that is used in trader and p2p), unless we change the interface of component.

Reply:
please add a comment or TODO for it.